javers-core/src/main/java/org/javers/core/diff/Diff.java

```diff
@@ -24,7 +25,7 @@
  *
  * @author bartosz walacik
  */
-public class Diff {
+public class Diff implements Serializable {
```

Comment:
thanks for contributing to JaVers,
what about Diff components? Change, Atomic, etc. Think we need a simple test which proofs that Diff serialization works.
You could add it to JaversDiffE2ETest